.github/scripts/trymerge.py

```diff
@@ -1806,6 +1808,12 @@ def post_comment(msg: str) -> None:
         gh_post_commit_comment(pr.org, pr.project, commit_sha, revert_msg)
         gh_update_pr_state(pr.org, pr.project, pr.pr_num)
 
+        reviewers = pr.get_approved_by()
```

Comment:
Let's move this into the if block below to avoid an unnecessary read request to Github when the author does have write permissions